src/components/AvatarStack/Avatars.tsx

```diff
@@ -1,66 +1,95 @@
-import { useState } from "react";
-import { UserCircleIcon } from "@heroicons/react/outline";
 import { SpaceMember } from "@ably-labs/spaces";
 import { colours } from "./utils/mockData";
 import { MAX_USERS_BEFORE_LIST } from "./utils/constants";
+import Surplus from "./Surplus";
+import { useState } from "react";
 import UserInfo from "./UserInfo";
 
 const SelfAvatar = () => (
-  <div className="group relative flex flex-col items-center group">
-    <UserCircleIcon className="absolute mt-2 h-8 w-8 opacity-80 text-white pointer-events-none" />
+  <div className="bg-orange-600 h-12 w-12 rounded-full flex items-center justify-center relative border-2 border-gray-200">
+    <p className="text-xs text-white">You</p>
     <div
-      className="bg-gradient-to-r from-green-500 to-blue-500
-                h-12 w-12 rounded-full mb-2"
-    ></div>
-    <div className="absolute top-14 invisible group-hover:visible px-4 py-2 bg-black rounded-lg text-white text-center">
-      You
-    </div>
+      className="bg-green-500 w-[10px] h-[10px] rounded-full absolute bottom-1 left-0 transform translate-y-1/2 translate-x-1/2"
+      id="status-indicator"
+    />
   </div>
 );
 
-const Avatars = ({ otherUsers }: { otherUsers: SpaceMember[] }) => {
+const OtherAvatars = ({ users }: { users: SpaceMember[] }) => {
   const [hoveredClientId, setHoveredClientId] = useState<string | null>(null);
-
   return (
     <>
-      {otherUsers
-        .slice(0, MAX_USERS_BEFORE_LIST)
-        .reverse()
-        .map((user, index) => {
-          const HORIZONTAL_SPACING_OFFSET = 40;
-          const rightOffset =
-            otherUsers.length > MAX_USERS_BEFORE_LIST
-              ? (index + 1) * HORIZONTAL_SPACING_OFFSET
-              : index * HORIZONTAL_SPACING_OFFSET;
-          return (
+      {users.map((user, index) => {
+        const HORIZONTAL_SPACING_OFFSET = 40;
+        const rightOffset =
+          users.length > MAX_USERS_BEFORE_LIST
+            ? (index + 1) * HORIZONTAL_SPACING_OFFSET
+            : index * HORIZONTAL_SPACING_OFFSET;
```

Comment:
I like to put staff like that in a util function. But that's just me, not necessary really 🙃